packages/website/docs/MultiSelect.md

```diff
@@ -1,23 +1,50 @@
-## Guidelines
+## Best Practices
 
-### General
+Use multi-select to allow user to select options from a **long list that can be filtered**. Multi-selects are more appropriate when the available list of options is very long or when the space is limited.
 
--   If space allows, and there are no more than 7 options in total, consider using [checkboxes](#/components/Checkbox) instead.
--   Keep option labels as short as possible in order to limit width to a minimum.
--   When an option is selected, it should be removed from the options list.
+Use alphanumerical order unless a more suited ordering rational applies (such as sizes or security levels).
 
-### Options Sorting
+Always include the ability to filter the list when there are over 20 options.
 
--   Always put the most common items at the top of the list. If no option is more common than the others, use alphanumeric or chronological order.
+## Labeling
 
-### Label
+Keep titles and labels short, **under one line**.
 
--   In most cases, you should include a label to identify the component.
+### Title
+
+The title is used to indicate the type of information to be provided.
+
+Use descriptive labels without action verbs.
+
+-   E.g. Use "Grocery items" instead of "Select all items for your grocery".
+
+### Labels
+
+Labels are used to name each option. The width of the input should allow to fully display the name of the chosen option.
+
+Use a consistent writing style for all options in the list.
+
+## Help text and instructions
+
+The placeholder text should indicate the type of information that must be selected with an action verb.
+
+-   E.g. Use "Select a condiment" instead of "Select an option".
+
+## Feedback and validation
+
+Allow the addition of custom values only when it doesn't increase the risk of failure and when the availability of all possible options in the list can not be ensured.
+
+Here some examples:
+
+-   When selecting countries to filter on: do not allow custom values.
+-   When filling a grocery list: allow custom values.
 
 ---
 
 ## Related Components
 
-[Single Select](#/components/SingleSelect)
+If your use case doesn't match these guidelines, consider using an alternate component instead.
 
-[Checkbox](#/components/Checkbox)
+-   [Multiline Box](#/form/MultilineBox) if real estate is not an issue, or if legibility of the selected option is critical.
```

Comment:
```suggestion
-   [Multiline box](#/form/MultilineBox) if real estate is not an issue, or if legibility of the selected option is critical.
```
@kpoulincoveo Pas sûre de comprendre: à la ligne 23, on dit que le label au complet devrait apparaître?

Reply:
@jlevesquecoveo yes but they are much smaller and stacked closely together while the multiline box has more space which increase legibility.